ostd/Cargo.toml

```diff
@@ -16,7 +16,7 @@ buddy_system_allocator = "0.9.0"
 cfg-if = "1.0"
 gimli = { version = "0.28", default-features = false, features = ["read-core"] }
 inherit-methods-macro = { git = "https://github.com/asterinas/inherit-methods-macro", rev = "98f7e3e" }
-xarray = { git = "https://github.com/asterinas/xarray", rev = "72a4067" }
+xarray = { git = "https://github.com/asterinas/xarray", rev = "1dad5d9" }
```

Comment:
Remember to also upgrade new version of xarray to `crates.io`, if you do any modification on xarray, since published ostd only relies on xarray on crates.io.